recipe/meta.yaml

```diff
@@ -102,23 +102,50 @@ source:
     # more fixes
     - patches/0010-fix-index-type-in-openmp-loop.patch
     - patches/0011-add-missing-headers.patch
+    # relax test: https://github.com/facebookresearch/faiss/commit/391601dc3f4793e10aaf9d21f5bbe2a94ecf4f50
+    # Relaxed a bit more due to the issue occasionally coming back
+    - patches/0012-relax_test_ivf_train_2level.patch
 
 build:
   number: 0
-  skip: true  # [cuda_compiler_version == "10.2"]
-  # always crashes
-  skip: true  # [win and cuda_compiler_version in ("11.0", "11.1")]
+  # some MKL issue for py 3.8 on osx-64
+  skip: true  # [py<38 or (py==38 and osx and x86_64)]
+  # Too many tests are failing on s390x.
+  skip: true  # [linux and (s390x or ppc64le)]
 
 requirements:
   build:
+    - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - patch  # [not win]
+    - m2-patch  # [win]
+    - cmake
+    - ninja
+    # libgomp is required for linux-64 alongside intel-omp.
+    - libgomp                 # [linux]
```

Comment:
Yeah I did not have this before, but I @skupr-anaconda suggested having a complete generic build and host sections. Without this, the build was a bit unstable on prefect. I don't know the reason. After I added these, I seem to stopped having problems of prefect occasionally failing to build.

Reply:
yes, that is weird, what you saying is bit of undefined behaviour

Reply:
Yeah, this should not be needed here, unless you are hardcoding paths to the `BUILD_ENV` somewhere in the build script(s). And I don't think it's needed when building locally, so it could be a flaky Prefect issue. But unfortunately we don't have the time to debug this ATM, so we might have to let this go for now.